academy/4-my-own-chain/ignitecli.md

```diff
@@ -127,22 +127,22 @@ If Vue.js is something new to you, check out the [Vue.js website](https://vuejs.
 
 </HighlightBox>
 
-If you look at the code that Starport generates, for instance in `./x/checkers/module.go`, you will often see comments like the following:
+If you look at the code that Ignite CLI generates, for instance in `./x/checkers/module.go`, you will often see comments like the following:
 
 ```go
-// this line is used by starport scaffolding # 1
+// this line is used by ignite scaffolding # 1
```

Comment:
@coldice @nooomski Example of comment left by Starport, probably not changed in the old version.

Reply:
Yeah we'll need to adjust these back to `this line is used by starport scaffolding # 1`

```suggestion
// this line is used by starport scaffolding # 1
```